SIS_import/sis_set_url.rb

```diff
@@ -342,6 +352,11 @@ def processTermCourses(mPathwayTermSet, esbToken)
 									result_json = setMPathwayUrl(esbToken, sisTermId, sectionParsedSISID, courseId)
 									if (!result_json.nil? && (result_json.has_key? "setLMSURLResponse"))
 										message = Time.new.inspect + " set url result for section id=#{sectionParsedSISID} with Canvas courseId=#{courseId}: result status=#{result_json["setLMSURLResponse"]["Resultcode"]} and result message=#{result_json["setLMSURLResponse"]["ResultMessage"]}"
+										# generate error message when there is a Failure status returned
```

Comment:
Either indentation is off or this section of code should be refactored into separate method.